modules/calc/src/main/java/com/opengamma/strata/calc/runner/SingleCalculationMarketData.java

```diff
@@ -7,20 +7,20 @@
 
 import java.time.LocalDate;
 
+import com.opengamma.strata.basics.market.MarketData;
 import com.opengamma.strata.basics.market.MarketDataKey;
 import com.opengamma.strata.basics.market.ObservableKey;
 import com.opengamma.strata.calc.marketdata.CalculationMarketData;
-import com.opengamma.strata.calc.marketdata.SingleCalculationMarketData;
 import com.opengamma.strata.collect.timeseries.LocalDateDoubleTimeSeries;
 
 /**
- * The default implementation of {@link SingleCalculationMarketData}.
+ * A set of market data providing access to a single scenario.
```

Comment:
"providing access to a single scenario" is a bit vague. Maybe "...providing data for a single scenario"